main.tf

```diff
@@ -0,0 +1,68 @@
+resource "null_resource" "default" {
+  triggers = {
+    id = "${lower(format("%v-%v-%v-%v", var.namespace, var.stage, var.name, "assets"))}"
```

Comment:
or better yet, let's add `attributes` as a new parameter to `tf_label` of type `list`. Use `join` to concatenate them with a `-`.